source/api/concepts/edit-sessions.rst

```diff
@@ -51,8 +51,11 @@ Edit Session Event
 ~~~~~~~~~~~~~~~~~~
 
 The ``EditSessionEvent`` is the way to hook into WorldEdit's changes in an efficient manner. It is fired as a part of
-``EditSession`` creation, and allows hooking into the extent stack at various steps as described above. You can
-subscribe to this event by registering with WorldEdit's event bus:
+``EditSession`` creation, and allows hooking into the extent stack at various steps as described above. The ``EditSessionEvent``
+is fired with the stages in the order of ``BEFORE_HISTORY``, then ``BEFORE_REORDER``, and at last ``BEFORE_CHANGE`` right before
+blocks in the world are modified.
```

Comment:
No, this is not correct. It is fired in the order already described, and like already said in the docs, at `EditSession` _creation_, not at modification time. This entire addition here should be removed.